tests/extension.test.js

```diff
@@ -69,9 +73,11 @@ describe("Extension: activate", () => {
         setWorkspaceFolders(repoPath);
         untildify.mockReturnValue(baseRepoPath);
         global.IS_CODESYNC_TEST_MODE = true;
-
+        pathUtilsObj = new pathUtils(repoPath, DEFAULT_BRANCH);
+        tabsRepo = pathUtilsObj.getTabsRepo();
         fs.mkdirSync(repoPath, {recursive: true});
         fs.mkdirSync(baseRepoPath, {recursive: true});
+        fs.mkdirSync(tabsRepo, {recursive:true});
```

Comment:
Why are we creating tabs directory separately when `createSystemDirectories();` should do that for us (used just below)